tests/jest/functional/stdout.spec.ts

```diff
@@ -29,9 +27,9 @@ const singleProjectDigraph = `digraph "io.snyk:single-project:jar:0.0.1-SNAPSHOT
 "org.springframework:spring-core:jar:5.3.20:compile" -> "org.springframework:spring-jcl:jar:5.3.20:compile" ; 
 } `;
 
-test('parseStdout single project', async (t) => {
-  const received = parseStdout(singleProjectStdout);
-  t.same(received, [singleProjectDigraph], 'contains single digraph');
+test('parseStdout single project', async () => {
+  const received = parseDigraphsFromStdout(singleProjectStdout);
```

Comment:
could also test your new function here? asserting the maven-dependency-plugin parsed is 2.8?